x-pack/plugins/security_solution/public/timelines/components/timeline/query_tab_content/index.tsx

```diff
@@ -344,6 +345,8 @@ export const QueryTabContentComponent: React.FC<Props> = ({
                   itemsCount: totalCount,
                   itemsPerPage,
                 })}
+                leadingControlColumns={[defaultControlColumn]}
```

Comment:
Consider creating constants (outside of the component's closure) for the arrays passed to the `leadingControlColumns` and `trailingControlColumns` props to ensure the same array references are passed as props.